Find a functionally equivalent code:android.net.ConnectivityManager.getLinkProperties:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Returns the IP information for a given network type.
 *
 * @param networkType the network type of interest.
 * @return a {@link LinkProperties} object describing the IP info
 * for the given networkType, or {@code null} if there is
 * no current default network.
 *
 * <p>This method requires the call to hold the permission
 * {@link android.Manifest.permission#ACCESS_NETWORK_STATE}.
 * {@hide}
 */

@@ -1,6 +1,6 @@
 {
     try {
-        return mService.getLinkProperties(networkType);
+        return mService.getLinkPropertiesForType(networkType);
     } catch (RemoteException e) {
         return null;
     }

